spec/fixtures/landing_page.yaml

```diff
@@ -1,12 +1,25 @@
+navigation_groups:
+- id: "Top Menu"
+  links:
+  - href: "/homepage"
+    text: "Homepage"
+    theme: "homepage"
```

Comment:
Is theme still being used? I think a comment said that it wasn't? If not, then this should be removed.

Reply:
Gone now!